src/assets/base.css

```
@@ -26,14 +26,30 @@ body {
     background 0.5s,
     background-color 0.5s;
   line-height: 1.6;
-  font-family: "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+  font-family:
+    "Inter",
```

Stylelint (17.11.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)